citrination_client/client.py

```diff
@@ -191,43 +191,43 @@ def predict(self, model_name, candidates):
     def _get_predict_url(self, model_name):
         return self.api_url + '/csv_to_models/' + model_name + '/predict'
 
-    def upload_file(self, file_path, data_set_id, root_path=None):
+    def upload(self, dataset_id, source_path, dest_path=None):
```

Comment:
This looks like an interface change but the `upload` method is still defined below.